ui/src/app/components/apps-panel/expanded-app.spec.tsx

```diff
@@ -165,7 +165,7 @@ describe('ExpandedApp', () => {
         expect(container).toBeInTheDocument();
 
         const actionButton = screen.getByRole('button', { name });
-        expectButtonElementEnabled(actionButton);
+        expectButtonElementDisabled(actionButton);
```

Comment:
oops.  how was this working?

Reply:
This is how the disabled pause button is structured on Test:
![Screenshot 2023-11-14 at 11 38 18 AM](https://github.com/all-of-us/workbench/assets/24514630/9e4f7d42-6f43-40b5-b602-4da6b216f178)

The getByRole was selecting the outer container, because it has the role of button, but you'll notice that it is has a cursor value of pointer, so that button was considered enabled. The inner component has cursor not-allowed, and that is what we would see in the UI since the inner container is the same size as the the outer container.

The new structure looks like this:
<img width="1185" alt="Screenshot 2023-11-14 at 11 51 54 AM" src="https://github.com/all-of-us/workbench/assets/24514630/c2f81bcc-91a0-4b0c-94e1-703b9f73c482">
This means that the element that is returned by getByRole also has the cursor not-allowed value.

Reply:
much better.  We should roll this out more generally.

Reply:
Thanks! Agreed!